server/api/applications.js

```diff
@@ -4,19 +4,21 @@ const {Applications} = require('../db/models')
 module.exports = router
 
 router.get('/', async (req, res, next) => {
-  if (!req.admin) {
+  if (!req.user) {
```

Comment:
@amesk3 and @ndittren 

Seems like Passport injects a `user` property onto the `req` object.